src/test/java/uk/gov/justice/digital/delius/service/PersonalCircumstanceServiceTest.java

```diff
@@ -49,16 +51,40 @@ public void personalCircumstancesOrderedNaturallyByPrimaryKeyReversed() {
     public void deletedRecordsIgnored() {
         Mockito.when(personalCircumstanceRepository.findByOffenderId(1L))
                 .thenReturn(ImmutableList.of(
-                        aPersonalCircumstance().toBuilder().personalCircumstanceId(1L).build(),
-                        aPersonalCircumstance().toBuilder().personalCircumstanceId(2L).softDeleted(1L).build(),
-                        aPersonalCircumstance().toBuilder().personalCircumstanceId(3L).build()
+                        aPersonalCircumstance().toBuilder().personalCircumstanceId(1L).startDate(LocalDate.now()).build(),
+                        aPersonalCircumstance().toBuilder().personalCircumstanceId(2L).startDate(LocalDate.now()).softDeleted(1L).build(),
+                        aPersonalCircumstance().toBuilder().personalCircumstanceId(3L).startDate(LocalDate.now()).build()
                 ));
 
         assertThat(personalCircumstanceService.personalCircumstancesFor(1L)
                 .stream().map(uk.gov.justice.digital.delius.data.api.PersonalCircumstance::getPersonalCircumstanceId)
                 .collect(Collectors.toList()))
                 .contains(1L, 3L);
 
+    }
+
+    @Test
+    public void personalCircumstancesHaveActiveFlag() {
+        LocalDate today = LocalDate.now();
+        LocalDate futureDate = LocalDate.now().plusDays(1);
+        LocalDate pastDate = LocalDate.now().minusDays(1);
+
+        Mockito.when(personalCircumstanceRepository.findByOffenderId(1L))
+                .thenReturn(ImmutableList.of(
+                        aPersonalCircumstance().toBuilder().personalCircumstanceId(1L).startDate(pastDate).endDate(today).build(),
+                        aPersonalCircumstance().toBuilder().personalCircumstanceId(2L).startDate(pastDate).endDate(futureDate).build(),
+                        aPersonalCircumstance().toBuilder().personalCircumstanceId(3L).startDate(pastDate).build()
+                ));
+
+        List<uk.gov.justice.digital.delius.data.api.PersonalCircumstance> personalCircumstances =
+            personalCircumstanceService.personalCircumstancesFor(1L);
+
+        assertThat(personalCircumstances.stream().filter(c -> c.getPersonalCircumstanceId().equals(1L))
+            .findFirst().get().getActiveFlag()).isEqualTo(false);
+        assertThat(personalCircumstances.stream().filter(c -> c.getPersonalCircumstanceId().equals(2L))
+            .findFirst().get().getActiveFlag()).isEqualTo(true);
+        assertThat(personalCircumstances.stream().filter(c -> c.getPersonalCircumstanceId().equals(3L))
+            .findFirst().get().getActiveFlag()).isEqualTo(true);
```

Comment:
Won't hurt but you probably don't need to test this since you've already tested the transformer?